booster-chat-app/src/read-models/MessageReadModel.ts

```diff
@@ -0,0 +1,32 @@
+import { ReadModel } from '@boostercloud/framework-core'
+import { UUID } from '@boostercloud/framework-types'
+import { Projects } from '@boostercloud/framework-core'
+import { User, Admin } from '../roles'
+import { UserProfile } from '../entities/UserProfile'
+import { Message, MessageType } from '../entities/Message'
+
+@ReadModel({
+  authorize: [User, Admin],
+})
+export class MessageReadModel {
+  public constructor(
+    public id: UUID,
+    readonly chatRoomId: UUID,
+    readonly body: string,
+    readonly messageType: MessageType,
+    readonly createdAt: string,
+    readonly createdBy: UserProfile
+  ) {}
+
+  @Projects(Message, 'chatRoomId')
+  public static onMessageCreated(message: Message): MessageReadModel {
```

Comment:
This is related to my previous comment about the Message class missing the Entity decorator. Is this code actually being called?